api/src/main/java/org/apache/iceberg/types/Types.java

```diff
@@ -70,6 +77,22 @@ public static Type fromTypeName(String typeString) {
       return TYPES.get(lowerTypeString);
     }
 
+    Matcher geometry = GEOMETRY_PARAMETERS.matcher(typeString);
+    if (geometry.matches()) {
+      String crs = geometry.group(1);
+      Preconditions.checkArgument(!crs.contains(","), "Invalid CRS: %s", crs);
```

Comment:
Why is this invalid? This should pass the string as-is. It is up to others to determine what is a valid CRS.

Reply:
I'll remove this check and pass the CRS string as-is, since it won't cause ambiguity for geometry types.